apps/dashboard/src/app/(dashboard)/company/page.tsx

```diff
@@ -0,0 +1,117 @@
+"use client"
+
+import { Company } from "@dotkomonline/types"
+import { Icon } from "@iconify/react"
+import {
+  Anchor,
+  Button,
+  ButtonGroup,
+  Card,
+  Group,
+  Skeleton,
+  Stack,
+  Table,
+  TableTbody,
+  TableTd,
+  TableTh,
+  TableThead,
+  TableTr,
+} from "@mantine/core"
+import { createColumnHelper, flexRender, getCoreRowModel, useReactTable } from "@tanstack/react-table"
+import { CompanyName } from "src/components/molecules/company-name/company-name"
+import { useCreateCompanyModal } from "src/modules/company/modals/create-company-modal"
+import { useCompanyAllQuery } from "src/modules/company/queries/use-company-all-query"
+import Link from "next/link"
+
+export default function CompanyPage() {
+  const { companies, isLoading: isCompaniesLoading } = useCompanyAllQuery()
+  const open = useCreateCompanyModal()
+
+  const columnHelper = createColumnHelper<Company>()
+  const columns = [
```

Comment:
Move out of component

Reply:
I refactored into hook in /modules/company/use-company-table.tsx. Not sure if this is an adequate location for the hook